content/v2.0/reference/flux/stdlib/experimental/geo/shapedata.md

```diff
@@ -0,0 +1,123 @@
+---
+title: geo.shapeData() function
+description: >
+  The `geo.shapeData()` function restructures data with existing latitude and longitude
+  fields to meet the requirements of the Geo package by renaming the existing latitude
+  and longitude fields to **lat** and **lon** and adding an **s2_cell_id** tag.
```

Comment:
How does function recognize existing latitude and longitude fields? Tags/fields that contain WGS 84 formatted data?

Does it make sense to say:
The `geo.shapeData()` function renames existing latitude and longitude fields(and/or tags?) to **lat** and **lon** (fields/tags?) and adds an **s2_cell_id** tag.

OR

The `geo.shapeData()` function restructures data by renaming...

Reply:
It doesn' t auto-detect the fields. You have to specify which fields are which.